.github/workflows/conformance.yml

```diff
@@ -242,10 +255,17 @@ jobs:
 
         echo "Running tests for Test${KIND_UPPER}Conformance/${KIND}/${NAME} ... "
 
+        if [ ${{ matrix.source-pkg }} != '' ]; then
+          SRC_PKG=${KIND}/${{ matrix.source-pkg }}
+        else
+          SRC_PKG=${KIND}/${NAME}  
+        fi
+
         set +e
         gotestsum --jsonfile ${{ env.TEST_OUTPUT_FILE_PREFIX }}_conformance.json \
           --junitfile ${{ env.TEST_OUTPUT_FILE_PREFIX }}_conformance.xml --format standard-verbose -- \
-          -p 2 -count=1 -timeout=15m -tags=conftests ./tests/conformance --run="Test${KIND_UPPER}Conformance/${NAME}"
+          -p 2 -count=1 -timeout=15m -tags=conftests ./tests/conformance --run="Test${KIND_UPPER}Conformance/${NAME}" -coverprofile=cover.out \
+          -covermode=set -coverpkg=github.com/dapr/components-contrib/internal/component/${{ matrix.source-pkg }}/...,github.com/dapr/components-contrib/${SRC_PKG}/...
```

Comment:
This is where `sourcePkg` is being used

Reply:
But in this case, `internal/component` may reference a folder that doesn’t exist. For example, if `matrix.source-pkg` is `azure/servicebusqueues` as in the example above, that folder doesn’t exist.

Reply:
Yeah, if a folder doesn't exist, that folder would be ignored in that case. And, coverage will factor in only present folders. Do you think there can be any problem in that case?

Reply:
No concerns with including additional folders, but the current code is not including all folders.

As per https://github.com/dapr/components-contrib/pull/2681#discussion_r1139750172 I would recommend making `sourcePkg` of type `string | string[]` to allow passing an array too, and then include the correct folders. `sourcePkg` could be an absolute package such as `github.com/dapr/components-contrib/foo`, since components may have shared code outside of `github.com/dapr/components-contrib/internal`

Reply:
That makes sense, Will update the `sourcePkg` to array of strings